xbmc/video/VideoDatabase.cpp

```diff
@@ -10282,3 +10517,34 @@ bool CVideoDatabase::SetVideoUserRating(int dbId, int rating, const MediaType& m
   }
   return false;
 }
+
+void CVideoDatabase::InitializeTypeTable()
+{
+  const char* types[] = { "Standard",
+                          "Extended",
+                          "Unrated",
+                          "Uncut",
+                          "Remastered",
+                          "Limited",
+                          "Special",
+                          "Director Cut",
+                          "The Final Cut",
+                          "Super Duper Cut",
+                          "Theatrical",
+                          "3D",
+                          "4K UHD",
+                          "IMAX",
+                          "Collector",
+                          "Ultimate Collector",
+                          "Criterion Collection",
+                          "10th Anniversary",
+                          "20th Anniversary",
+                          "25th Anniversary",
+                          "30th Anniversary",
+                          "40th Anniversary",
+                          "50th Anniversary",
+  };
+
+  for (const char* type : types)
+    m_pDS->exec(PrepareSQL("INSERT INTO type VALUES(NULL, '%s')", type));
```

Comment:
I had the following in mind,
```suggestion
    m_pDS->exec(PrepareSQL("INSERT INTO type (name) VALUES('%s')", type));
```

Reply:
Thanks for this trick. This makes the code simpler. For me I always use VALUES for INSERT clause to make it more clear, and there is no performance difference.